### PR TITLE
Task nxp 26962 arender polymer 3

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: addons/nuxeo-arender/elements/nuxeo-arender-blob-diff-button.js [hunk 1]
```diff
@@ -66,57 +67,53 @@
...
     </style>
 
-    <nuxeo-operation id="aRenderDiffOp"
-                     op="Document.ARenderGetDiffUrl">
+    <nuxeo-operation id="aRenderDiffOp" op="Document.ARenderGetDiffUrl">
     </nuxeo-operation>
 
     <div class="action" on-tap="_openDialog">
-      <paper-icon-button noink id="aRenderblobDiffButton" icon="[[icon]]"></paper-icon-button>
+      <paper-icon-button noink="" id="aRenderblobDiffButton" icon="[[icon]]"></paper-icon-button>
```
Comment (nmpcunha) [inline addons/nuxeo-arender/elements/nuxeo-arender-blob-diff-button.js]: Should we remove `=""`?

File: addons/nuxeo-arender/elements/nuxeo-arender-blob-diff-button.js [hunk 2]
```diff
@@ -66,57 +67,53 @@
...
     </div>
     <nuxeo-tooltip for="aRenderblobDiffButton">[[i18n('documentDiffButton.blob.tooltip')]]</nuxeo-tooltip>
 
-    <nuxeo-dialog on-iron-overlay-closed="_closeDialog" id="aRenderBlobDiffDialog" with-backdrop>
+    <nuxeo-dialog on-iron-overlay-closed="_closeDialog" id="aRenderBlobDiffDialog" with-backdrop="">
```
Comment (nmpcunha) [inline addons/nuxeo-arender/elements/nuxeo-arender-blob-diff-button.js]: Same here

File: addons/nuxeo-arender/elements/nuxeo-arender-blob-diff-button.js [hunk 3]
```diff
@@ -66,57 +67,53 @@
...
       <template>
         <iframe src="[[_aRenderUrl]]"></iframe>
-        <paper-icon-button id="close-icon" icon="nuxeo:clear" on-click="_closeDialog" noink></paper-icon-button>
+        <paper-icon-button id="close-icon" icon="nuxeo:clear" on-click="_closeDialog" noink=""></paper-icon-button>
```
Comment (nmpcunha) [inline addons/nuxeo-arender/elements/nuxeo-arender-blob-diff-button.js]: And here!

File: addons/nuxeo-arender/elements/nuxeo-arender-blob-diff-button.js (hunk 1 above)
Comment (nelsonsilva) [reply addons/nuxeo-arender/elements/nuxeo-arender-blob-diff-button.js]: This is produced by polymer modulizer during the migration. I'd rather cleanup the whole codebase afterwards

**Comment by Gabez0r** [comment]
👍

**Pull request merged**